OpenSim/Actuators/Test/testMuscles.cpp

```diff
@@ -72,35 +72,117 @@ static const double Activation0     = 0.01,
                     ShutteDelpActivation1 = 7.6,    
                     ShutteDelpActivation2 = 2.5;
 
-/*
-This function completes a controlled activation, controlled stretch simulation 
-of a muscle. After the simulation has completed, the results can be 
-tested in a number of different ways to ensure that the muscle model is 
-functioning
-
-@param aMuscle  a path actuator
-@param startX   the starting position of the muscle anchor. I have no idea
-                why this value is included.
-@param act0     the initial activation of the muscle
-@param motion   the forced stretch of the simulation
-@param control  the activation control signal that is applied to the muscle
-@param printResults print the osim model associated with this test.
+// Helper functions for tests
```

Comment:
nit: It's self-evident that these are helper functions for the tests, so I'd say this comment is unnecessary.